cluster/etcd.go

```diff
@@ -156,3 +171,14 @@ func (c *Cluster) getBackupImage() string {
 	}
 	return util.GetDefaultRKETools()
 }
+
+func IsLocalSnapshot(name string) bool {
+	// name is fmt.Sprintf("%s-%s%s-", cluster.Name, typeFlag, providerFlag)
```

Comment:
will it take care of the legacy snapshots taken prior to 2.2?

Reply:
You might want to make the reverse check. IsS3Snapshot.

Reply:
Legacy snapshots are not affected in this case.